### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by webknjaz** [re-actors/alls-green#8] [Announcements]
## What's New

This release implements posting check details as a summary on the workflow overview page.

**Full Diff**: https://github.com/re-actors/alls-green/compare/v1.1.0...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/re-actors/alls-green/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by ssbarnea** [comment]
Looks like a nice addition. I hope it still does not post anything if all green.

**Reply by webknjaz** [reply]
It does.
